doc/man3/PKCS12_add_safe.pod

```diff
@@ -0,0 +1,63 @@
+=pod
+
+=head1 NAME
+
+PKCS12_add_safe, PKCS12_add_safes - Create and add objects to a PKCS#12 structure
+
+=head1 SYNOPSIS
+
+ #include <openssl/pkcs12.h>
+
+ int PKCS12_add_safe(STACK_OF(PKCS7) **psafes, STACK_OF(PKCS12_SAFEBAG) *bags,
+                    int safe_nid, int iter, const char *pass);
+ PKCS12 *PKCS12_add_safes(STACK_OF(PKCS7) *safes, int p7_nid);
+
+=head1 DESCRIPTION
+
+PKCS12_add_safe() creates a new PKCS7 contentInfo containing the supplied
+B<PKCS12_SAFEBAG>s and adds this to a set of PKCS7 contentInfos. Its type
```

Comment:
I'm not sure I like B<PKCS12_SAFEBAG>s but haven't thought of an alternative just yet.